torchtitan/val.py

```diff
@@ -150,12 +161,12 @@ def validate(
     if loss:
         del loss
 
-    if os.path.exists(end_of_validation_path) and dp_rank == 0:
-        os.remove(end_of_validation_path)
-        logger.info("removed the store file")
-    else:
-        logger.info("no store file exists")
-    del val_data_iterator, data_loader, end_of_validation_store
+    # if os.path.exists(end_of_validation_path) and dp_rank == 0:
```

Comment:
If we aren't using the store for signalling end of validation, we can remove these comments